Clicker/ViewControllers/GroupControlsViewController.swift

```diff
@@ -0,0 +1,235 @@
+    let infoViewHeight: CGFloat = 20
+    let infoViewTopPadding: CGFloat = 28
+    let infoViewHorizontalPadding: CGFloat = 30
+    let collectionViewTopPadding: CGFloat = 16
+    let collectionViewBottomPadding: CGFloat = 24
+    let collectionViewHeight: CGFloat = 218
+    let exportButtonWidthScaleFactor: CGFloat = 0.43
+    let exportButtonHeight: CGFloat = 47
+    let exportButtonBorderWidth: CGFloat = 1
+    let exportButtonTopPadding: CGFloat = 12
+    let navigtionTitle = "Group Controls"
+    let backImageName = "back"
+    let attendanceLabelText = "Attendance"
+    let exportButtonTitle = "Export"
+
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .lightContent
+    }
+
+    init(session: Session, pollsDateAttendanceArray: [PollsDateAttendanceModel], numMembers: Int) {
+        super.init(nibName: nil, bundle: nil)
+        self.session = session
+        self.pollsDateAttendanceArray = pollsDateAttendanceArray
+        self.numMembers = numMembers
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .clickerBlack1
+
+        setupNavBar()
+        setupViews()
+        setupConstraints()
+    }
+
+    private func setupNavBar() {
+        navigationController?.setNavigationBarHidden(false, animated: false)
+        // REMOVE BOTTOM SHADOW
+        navigationController?.navigationBar.setBackgroundImage(UIImage(), for: UIBarMetrics.default)
+        navigationController?.navigationBar.shadowImage = UIImage()
+
+        navigationTitleView = NavigationTitleView()
+        navigationTitleView.configure(primaryText: navigtionTitle, secondaryText: session.name)
+        self.navigationItem.titleView = navigationTitleView
+
+        let backImage = UIImage(named: backImageName)?.withRenderingMode(.alwaysOriginal)
+        self.navigationItem.leftBarButtonItem = UIBarButtonItem(image: backImage, style: .done, target: self, action: #selector(goBack))
+    }
+
+    private func setupViews() {
+        infoView = GroupControlsInfoView()
+        let numPolls = pollsDateAttendanceArray.reduce(0) { (result, pollsDateAttendanceModel) -> Int in
+            return result + pollsDateAttendanceModel.model.polls.count
+        }
+        infoView.configure(numMembers: numMembers, numPolls: numPolls, code: session.code)
+        view.addSubview(infoView)
+
+        attendanceLabel = UILabel()
+        attendanceLabel.text = attendanceLabelText
+        attendanceLabel.font = ._21SemiboldFont
+        attendanceLabel.textAlignment = .center
+        attendanceLabel.textColor = .white
+        view.addSubview(attendanceLabel)
+
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.showsVerticalScrollIndicator = false
+        collectionView.showsHorizontalScrollIndicator = false
+        collectionView.bounces = false
```

Comment:
either bounces is default on or delete off so one of these should be deleted lol